pkg/cloud/baremetal/actuators/machine/actuator.go

```diff
@@ -910,7 +910,8 @@ func (a *Actuator) requestPowerOff(ctx context.Context, baremetalhost *bmh.BareM
 		return &machineapierrors.RequeueAfterError{RequeueAfter: time.Second * 5}
 	}
 
-	baremetalhost.Annotations[requestPowerOffAnnotation] = ""
+	// By default we issue a hard reboot for remediation purposes
```

Comment:
It's not really the default, though. There's no way to change the behavior from this point, is there? So, it's "always issue a hard reboot..."

Reply:
Agreed. Will fix.